src/main/java/com/corpo/coliseum/domain/entity/Corporation.java

```diff
@@ -9,14 +9,17 @@
 import javax.persistence.Entity;
 import javax.persistence.ManyToMany;
 import javax.persistence.Table;
+import java.io.Serializable;
 import java.util.List;
 
 @Entity
 @Table(name = "corporation")
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-public class Corporation extends PanacheEntity {
+public class Corporation extends PanacheEntity implements Serializable {
+
+    private static final long serialVersionUID = 42L;
```

Comment:
what is the point of fixing this number?

is it necessary in this context ?

Reply:
https://stackoverflow.com/questions/285793/what-is-a-serialversionuid-and-why-should-i-use-it

"However, it is strongly recommended that all serializable classes explicitly declare serialVersionUID values, since the default serialVersionUID computation is highly sensitive to class details"